src/nonprintable_notation.rs

```diff
@@ -0,0 +1,11 @@
+/// How to print non-printable characters with
+/// [crate::config::Config::show_nonprintable]
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq)]
+pub enum NonprintableNotation {
```

Comment:
I forgot one thing! We need to make the new enum `#[non_exhaustive]`. Otherwise adding more notations will require a major semver version bump.

```suggestion
#[non_exhaustive]
pub enum NonprintableNotation {
```

You don't have to add it though. I can add it before I merge this. I plan on merging it early next week, unless something else comes up.